src/firefly/js/visualize/saga/ActiveRowToImageWatcher.js

```diff
@@ -97,6 +99,7 @@ const isImageAitoff= (plot) => (isImage(plot) && plot.projection.isWrappingProje
 function shouldCenterOnTableRow(pv, wp, force) {
     const plot= primePlot(pv);
     if (!plot) return false;
+    if (pv?.plotViewCtx.ignorePanByTableRow) return false;
```

Comment:
I think you can use the constant here as a key to prevent name changes breaking the code:

```suggestion
    if (pv?.plotViewCtx[PlotAttribute.IGNORE_PAN_BY_TABLE_ROW]) return false;
```